pylsp/config/schema.json

```
@@ -353,6 +353,14 @@
       "type": ["string", "null"],
       "default": null,
       "description": "Executable to run pylint with. Enabling this will run pylint on unsaved files via stdin. Can slow down workflow. Only works with python3."
+    },"pylsp.plugins.rope_autoimport.enabled": {
+      "type": "boolean",
+      "default": false,
+      "description": "Enable or disable the plugin."
+    },"pylsp.plugins.rope_autoimport.memory": {
```

```suggestion
    },
    "pylsp.plugins.rope_autoimport.enabled": {
      "type": "boolean",
      "default": false,
      "description": "Enable or disable autoimport."
    },
    "pylsp.plugins.rope_autoimport.memory": {
```

Please don't forget to update CONFIGURATION.md after this.